iib/web/static/api_v1.yaml

```diff
@@ -275,11 +275,15 @@ paths:
           description: services and workers are working as expected.
           content:
             application/json:
+              schema:
+                type: object
```

Comment:
Thank you for adding the schema, I really appreciate the effort!
However the schema was nested and there were also properties set for given object which is still missing.
Could you add correct properties for both responses (200 and 500)?

Thank you!